src/dotnet/Chat.Service/Db/DbReaction.cs

```diff
@@ -0,0 +1,41 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace ActualChat.Chat.Db;
+
+[Table("Reactions")]
+[Index(nameof(ChatId), nameof(EntryId), nameof(AuthorId))]
+public class DbReaction : IHasId<string>
+{
+    public DbReaction() { }
+    public DbReaction(Reaction model) => UpdateFrom(model);
+
+    [Key] public string Id { get; set; } = null!;
+    public string AuthorId { get; set; } = "";
+    public string ChatId { get; set; } = "";
```

Comment:
May be it make sense to combine ChatId and EntryId to ChatEntryId property like you did for DbTextEntryAttachement?

Reply:
done